src/components/MoviesSection/SortDropdown.tsx

```diff
@@ -28,11 +31,17 @@ export const SortDropdown: FC = () => {
     e.stopPropagation();
     if (button.textContent) {
       setSortName(button.textContent);
+      let sortField = '';
       if (button.textContent === SortName.HIGHEST_RATING) {
-        // todo: add logic in next tasks
+        sortField = 'vote_average';
       } else if (button.textContent === SortName.NEWEST) {
-        // todo: add logic in next tasks
+        sortField = 'release_date';
```

Comment:
Let's use enum for sort types

Reply:
Added enum in commit e9860952613603eb2cf3095f28c180d207b9668c